interface/forms/eye_mag/view.php

```diff
@@ -2660,14 +2660,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
                                                         <br />
                                                         <table>
                                                             <tr>
-                                                                <td style="text-align:center;"><?php echo xlt('R{{right}}'); ?></td>
+                                                                <td class='text-center'><?php echo xlt('R{{right}}'); ?></td>
                                                                 <td style="border-right:1pt solid black;border-bottom:1pt solid black;text-align:right;">
```

Comment:
Convert the style tag of this `<td>` element to the following classes: `border-right border-bottom border-dark text-right`